DAL/SQLHelper.cs

```diff
@@ -533,17 +533,18 @@ public void UpdateContentItem(StructuredContent_ContentType content_type, dynami
                     if (content_field.is_system == false)
                     {
                         sbUpdateContentItem.Append(", [" + content_field.column_name + "]");
-
                         sbUpdateContentItem.Append("=");
 
-                        object value = content_item[content_field.column_name];
-
-                        if (value == null)
+                        var field_value = content_item[content_field.column_name];
```

Comment:
If `value` is renamed here, it needs to be renamed everywhere it is referenced...